examples/vulnerabilities-connector/app/ProviderConfig.go

```diff
@@ -0,0 +1,45 @@
+package app
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/synqly/go-sdk/client/engine"
+	"github.com/synqly/go-sdk/examples/common"
+
+	mgmt "github.com/synqly/go-sdk/client/management"
+)
+
+type ProviderConfiguration struct {
+	AccountName        string
+	IntegrationRequest *mgmt.CreateIntegrationRequest
+	Name               string
+	Tenant             *common.Tenant
+}
+
+func (p *ProviderConfiguration) New(config Configuration, providerConfig mgmt.ProviderConfig, providerName string) error {
+	p.AccountName = fmt.Sprintf("Zenith Systems - %s", providerName)
```

Comment:
can you make the account name `"Zenith Systems - "` a const? that way you can use it here and in the cleanup